CSS-Fonts (#3044)
Codecademy/docs: content/css/concepts/typography/terms/font-face/font-face.md

  src: url(' ') /* Link to font */;
}
```
There are different types of Font Formats that we use - WOFF,WOFF2(more compressed version of WOFF) EOT ,TTF and SVG.

**Note:** The name of the file in which the font is stored, must end in a supported font format(e.g. **custom_font.woff**,**custom_font.woff2**, **custom_font.ttf**,**custom_font.svg**,**custom_font.eot**).

> If we need our font to load faster then WOFF is the best(this format is supported in most of the browsers).

Reviewer: I think font formats should be covered in greater detail in a different entry. Maybe in General File Formats (cc: @caupolicandiaz if you concur)

I'm thinking that it can go in a separate header ## Font Formats and be listed in table format. Then, we can link out to it here.

## Example 1
The below code snippet uses the `font-family` property and the style `superFont` to give the text a unique look 

> The below code snippet uses the `font-family` property and the style `superFont` to give the text a unique look 

Reviewer: Can you remove one of these lines

  src: url(' ') /* Link to font */;
}
```
There are different types of Font Formats that we use - WOFF,WOFF2(more compressed version of WOFF) EOT ,TTF and SVG.

**Note:** The name of the file in which the font is stored, must end in a supported font format(e.g. **custom_font.woff**,**custom_font.woff2**, **custom_font.ttf**,**custom_font.svg**,**custom_font.eot**).
If we need our font to load faster then WOFF is the best(this format is supported in most of the browsers).

> ## Example 1

Reviewer 2: [suggestion] Readers might wonder what this code snippet is doing, so consider adding background information before it. 
```suggestion
## Example 1
The below code snippet uses the `font-family` property and the style `superFont` to give the text a unique look 
```

  font-family: superFont;
}
```

> **Note:**  Another think to note while using any font is to check for browser support. If the fomrat that we use is not supported, it will just roll back to default system font. 

Reviewer 2: [Proofreading] 
```suggestion
**Note:**  Another thing to note while using any font is to check for browser support. If the format that we use is not supported, it will just roll back to the default system font. 
```

  font-family: superFont;
}
```

> **Note:**  Another thing to note while using any font is to check for browser support. If the format that we use is not supported, it will just roll back to the default system font. 

Reviewer: ```suggestion

> **Note:**  While using any font, check for browser support. If the format is not supported, it will use the default system font. 
```
